test/types/optionSelect.test.tsx

```diff
@@ -58,4 +61,32 @@ describe('optionSelect', () => {
     expect(tester.text()).not.toContain('Yes');
     expect(tester.text()).not.toContain('2 Values');
   });
+
+  it('Shows search for large data sets, or on command', async () => {
+    // tslint:disable no-magic-numbers
+    const states = Array(50).map((_v: any) => ({
+        name: fakeTextShort(),
+        value: faker.random.uuid(),
+      }))
+      , fieldConfig = { field, options: states, optionType: 'state', type };
+
+    function getPropsFor (overrides: Partial<IFieldConfig> = {}) {
```

Comment:
Why not `getProps`, `getOverriddenProps`, `getPropsWithOverrides`? I find prepositions at the end of function names usually point to 1. an unneeded word or 2. a missing noun